how-to/troubleshoot/update-verify-logs.rst

```diff
@@ -1,47 +1,239 @@
+    HTTP request sent, awaiting response... 
+      HTTP/1.1 200 OK
+      Server: nginx
+      Date: Sat, 08 Feb 2025 00:47:17 GMT
+      Content-Type: text/xml; charset=utf-8
+      Content-Length: 769
+      Vary: Accept-Encoding
+      Rule-ID: 25
+      Rule-Data-Version: 1231
+      Strict-Transport-Security: max-age=31536000;
+      X-Content-Type-Options: nosniff
+      Content-Security-Policy: default-src 'none'; frame-ancestors 'none'
+      X-Proxy-Cache-Status: MISS
+      Via: 1.1 google
+      Cache-Control: public,max-age=90
+      Alt-Svc: clear
+    Length: 769 [text/xml]
+    Saving to: ‘update.xml’
+    
+         0K                                    100% 17.0M=0s
+    
+    2025-02-08 00:47:17 (17.0 MB/s) - ‘update.xml’ saved [769/769]
+    
+    Got this response:
+    <?xml version="1.0"?>
+    <updates>
+        <update actions="showURL" appVersion="142.0" buildID="20250207232657" detailsURL="https://www.mozilla.org/en-CA/firefox/142.0/releasenotes/" displayVersion="142.0 Beta 5" openURL="https://www.mozilla.org/en-CA/firefox/142.0beta/whatsnew/?oldversion=%OLD_VERSION%&amp;utm_medium=firefox-desktop&amp;utm_source=update&amp;utm_campaign=142.0beta" type="minor">
+            <patch type="complete" URL="https://ftp.stage.mozaws.net/pub/firefox/candidates/142.0b5-candidates/build1/update/linux-x86_64/en-CA/firefox-142.0b5.complete.mar" hashFunction="sha512" hashValue="657e9e6b802bd879428dd608924198018f690db28afedbf42b964835571ae4dcd692559f93c4da82ac532b74e97bf9d25fb4041c1e8df63717c1e26fd28c37b9" size="69608506"/>
+        </update>
+    </updates>
+    
+    Retrieving 'https://ftp.stage.mozaws.net/pub/firefox/candidates/142.0b5-candidates/build1/update/linux-x86_64/en-CA/firefox-142.0b5.complete.mar' from cache...
+
+Log summary
+^^^^^^^^^^^
+
+Update verify performs some *very* basic log analysis and summary which is printed after all tests have been run. For a successful run, we expect to see:
+
+.. code-block:: text
+
+    Scanning log for failures and warnings
+    --------------------------------------
+    -------------------------
+    All is well
+
+A run with failures may, of course, have varied output, but one example of failure out is:
+
+.. code-block:: text
+
+    Scanning log for failures and warnings
+    --------------------------------------
+    TEST-UNEXPECTED-FAIL: update status was not successful: 
+    check_updates returned failure for Linux_x86-gcc3 downloads/firefox-135.0b9.tar.xz vs. downloads/firefox-136.0b2.tar.xz: 1
+    TEST-UNEXPECTED-FAIL: update status was not successful: 
+    check_updates returned failure for Linux_x86-gcc3 downloads/firefox-135.0b9.tar.xz vs. downloads/firefox-136.0b2.tar.xz: 1
+    -------------------------
+    This run has failed, see the above log
+
+The square-bracketed string (eg: ``[135.0 bn complete]``) can be searched for in the log to find the full output from the test that failed.
```

Comment:
Should there be a square-bracketed string in the example output above?

Reply:
I think so...something seems to have gone wrong in my copy paste. I'll fix this up one way or the other.